.github/workflows/release-publish.yaml

```diff
@@ -0,0 +1,56 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+permissions:
+  contents: write
+
+jobs:
+  source-wheel:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+          cache: "pip"
```

Comment:
This specific `cache` line is not on craft-archives' version; should it be there too?

Reply:
Probably, but there's no urgency to it. It speeds up the workflow slightly since it'll keep the cache for build and twine, but looking at [the last run](https://github.com/canonical/craft-archives/actions/runs/5124318067/jobs/9216009652) that's 4 seconds.